pulpcore/cli/common/generic.py

```diff
@@ -112,6 +112,19 @@ def load_json_callback(
         return json_object
 
 
+def _base_path_callback(
+    ctx: click.Context, param: click.Parameter, value: Optional[str]
+) -> Optional[str]:
+    if value is not None:
+        if ctx.info_name == "show":
+            entity_ctx: PulpEntityContext = ctx.find_object(PulpEntityContext)
+            entity_ctx.entity = {"base_path": value}
+        elif ctx.info_name == "list":
+            param.name = "base_path__icontains"
+            param.expose_value = True
```

Comment:
I'd say, if you want a base_path__icontains filter, that make it a --base-path-contains option.